src/runtime_src/xrt/device/hal2.cpp

```diff
@@ -123,7 +123,7 @@ void
 device::
 acquire_cu_context(const uuid& uuid,size_t cuidx,bool shared)
 {
-#if 0
+#if 1
```

Comment:
Enabling this implies we are confident mailbox works.

Reply:
Yes it does, but we are still protected by XCL_MULTIPROCESS_MODE which we should remove once MailBox is solid.

Reply:
No, we are not protected by the env var for acquire/release_cu_context.  Enabling this code implies that xclOpen/CloseContext will be called always and reference counting of xclbin better be working.  I am okay enabling it after Max's changes yesterday.